azure/onboarding/byok8s/manualsteps/README.md

```diff
@@ -0,0 +1,59 @@
+# Documentation for Manually Creating an AKS Cluster for RSC Exocompute
+This documentation outlines the steps to manually create an AKS cluster and associated resources on Azure to be used as n exocompute cluster with RSC
+
+## Prerequisites:
+1. Azure CLI configured with necessary permissions.
```

Comment:
Should we explcitly mention that user needs to run `az login` command and select appropriate subscription before running following commands?

Reply:
sure will add that